### PR TITLE
Jira 2.4 docs update

### PR DESCRIPTION
**Pull request by justinegeffen**
Updated to align with 2.3 changes. Grammar, structure, trailing spaces.

### CODE REVIEW

**Review by jfrerich — APPROVED** [review]
Thanks again @justinegeffen! Having someone look over the docs and create consistency across the entire document is invaluable!

**Comment by justinegeffen** [comment]
@aaronrothschild, these are the same changes as for 2.3, so basically just a first-pass cleanup. LMK if you have any feedback - we can also open a new PR for any major changes.

File: docs/admininstrator-guide/notification-management.md [hunk 1]
```diff
@@ -6,43 +6,41 @@ description: Centrally view all notification subscriptions in your system
...
-![This is a channel notification of a new bug that was created in Jira](../.gitbook/assets/image%20%281%29.png)
-
-When any webhook event is received from Jira - the plugin reviews all the notification subscriptions and if it matches a rule - it will post a notification to the channel.  If there are no subscription matches, the webhook event is discarded. 
+Notifications and webhooks can be used together or you can opt for one of them.
 
-The notifications and metadata shown in a channel are not protected by Jira permissions.  Anyone in the channel can see what is posted to the channel -  if they click on the issue link, Jira will not let the user see further details if they do not have the appropriate permission.  
+![This is a channel notification of a new bug that was created in Jira](../.gitbook/assets/image%20%281%29.png)
 
-## What is a Notification Subscription?
+When any webhook event is received from Jira the plugin reviews all the notification subscriptions. If it matches a rule it will post a notification to the channel. If there are no subscription matches, the webhook event is discarded.
```
Comment (aaronrothschild) [inline docs/admininstrator-guide/notification-management.md:15]: good call out

**Review by aaronrothschild — APPROVED** [review]
LGTM!  Thanks!

**Pull request merged**